content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,65 @@
+
+Refer to the [installation option change page](/docs/reference/config/installation-options-changes/) to view the complete list of changes.
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.
+- **Introduced** IPV6 alpha support for Kubernetes clusters.
+
+## `istioctl` and `kubectl`
+
+- **Graduated** `istioctl verify-install` out of experimental.
+- **Improved** `istioctl verify-install` to validate if a given Kubernetes environment meets Istio's prerequisites.
+- **Added** auto-completion support to `istioctl`.
+- **Added** `istioctl experimental dashboard` to allow users to easily open the web UI of any Istio addons.
+- **Added** `istioctl x` alias to conveniently run `istioctl experimental` command.
+- **Improved** `istioctl version` to report both Istio control plane and `istioctl` version info by default.
+- **Improved** `istioctl validate` to validate Mixer configuration and supports deep validation with referential integrity.
+
+## Traffic Management
+
+- **Improved** locality based routing in multicluster environments.
+- **Improved** outbound traffic policy in `ALLOW_ANY` mode. Traffic for unknown HTTP/HTTPS hosts on an existing port will be forwarded as is. Unknown traffic will be logged in Envoy access logs.
+- **Added** support for sending traces to Datadog.
+- **Added** support for setting HTTP idle timeouts to upstream services.
+- **Improved** Sidecar support for NONE mode (without iptables).
+- **Added** support for configuring the paths for Istio mutual TLS certificates.
+- **Added** ability to configure the DNS refresh rate for sidecar Envoys, to reduce the load on the DNS servers.
+
+## Security
+
+- **Added** Kubernetes health check prober rewrite per deployment via `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in the `PodSpec` annotation.
+- **Added** Support for [PKCS 8](https://en.wikipedia.org/wiki/PKCS_8) private keys for workloads, enabled by the field `pkcs8-keys` on Citadel.
+- **Fixed** SAN field in workload certificates is set as `critical`.
+- **Improved** Added Jitter to Citadel Agent for sending CSR requests.
+- **Improved** Extended the lifetime of self-signed Citadel root certificates to 10 years.
+- **Improved** JWT public key fetching logic to make it more resilient to network failure.
+
+## Telemetry
+
+- **Added** Envoy metrics support to include metrics by prefixes, suffixes and regular expressions.
+- **Changed** Prometheus generated traffic is excluded from metrics.
+
+## Policy
+
+- **Fixed** TCP Policy enforcement.
+
+## Configuration Management
+
+- **Added** ControlZ support to visualize the state of the MCP Server in Galley.
+- **Enabled** validation of more Mixer CRDs in Galley validation webhook.
+- **Added** support for specifying connection metadata for MCP dial-out mode in Galley.
```

Comment:
Sorry hit resolved on the wrong comment. this is still needed: @ozevren Is there a link to provide here in user docs?

Reply:
We have command-line flags documented, but not a more comprehensive doc. In either case, this is a very rough-cut alpha-like feature at this point.

Reply:
https://preliminary.istio.io/docs/reference/commands/galley/#galley-server

Reply:
discussed with @ozevren offline, think this would not be used by users directly, and we should remove it.